content/07.pathogenesis.md

```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
 
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis
 
-[Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
+Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
+Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
+
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
+A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7]. 
+Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 
+Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10.1016/S0140-6736(20)30251-8]. 
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7].
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
```

Comment:
It refers to the 3 genome sequence papers - One Lancet and Two Natures

Reply:
I have removed the immunology related discussion from this sub-section to avoid redundancy. We will add those and other suggested papers in the "Immune Response to SARS-CoV-2" sub-section.

Reply:
Briefly corresponded with @rando2 . This was good to go for now & we will continue to revise as more is learned 👍